frontend/src/views/ExportCenter/DataExportIssueDataTable/DataExportIssueDataTable.vue

```diff
@@ -187,20 +195,4 @@ watch(
     }
   }
 );
-
-const highlights = computed(() => {
-  if (!props.highlightText) {
```

Comment:
Seems we can remove the `highlightText` props for `DataExportIssueDataTable.vue`

Reply:
Done